### PR TITLE
Google Tag Manager integration

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Adds GTM to the site
- Switched GA calls to happen via GTM
- GTM (and GA) is loaded by default but linking to GA is configured in GTM based on cookie acceptance (sign off of this TBC)

### CODE REVIEW

File: src/lib/actions.js [hunk 1]
```diff
@@ -229,11 +230,12 @@ export const setLanguage = store.action((state, language) => {
 });
 
 export const loadAnalyticsScript = store.action(() => {
-  const {g4ScriptLoaded} = store.getState();
-  if (!g4ScriptLoaded && isProd) {
-    loadScript(`https://www.googletagmanager.com/gtag/js?id=${ids.GA4}`, null);
+  const {gtmScriptLoaded} = store.getState();
+  if (!gtmScriptLoaded && !isProd) {
```
Comment (tunetheweb) [inline src/lib/actions.js:234]: Change this back to `isProd` before merging:

```suggestion
  if (!gtmScriptLoaded && isProd) {
```

Or alternatively we can remove this, now we have staging properly set up.

File: src/site/_includes/partials/analytics.njk [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-gtag('js', new Date());
+window.dataLayer.push({'gtm.start': new Date().getTime(),event:'gtm.js'});
+window.dataLayer.push({'user_agent': navigator.userAgent});
+//Restrict GTM variables:
+dataLayer.push({'gtm.blacklist': ['html', 'd', 'jsm', 'j']});
+function gtag(){dataLayer.push(arguments)};
 {% if (site.env != 'prod') %}
   {# In development, log debug events to the console instead of sending them to GA #}
-  window.gtag = console.debug;
+  //window.gtag = console.debug;
```
Comment (tunetheweb) [inline src/site/_includes/partials/analytics.njk:10]: Uncomment this before merging.

Or alternatively remove this, now we have staging setup.

File: src/lib/store.js [hunk 3]
```diff
@@ -49,7 +49,8 @@ const initialState = {
   // cookie policy.
   // We automatically accept cookies in dev and test environments so the cookie
   // banner doesn't interfere with tests.
-  cookiePreference: isProd ? null : 'accepts',
+  // cookiePreference: isProd ? null : 'accepts',
+  cookiePreference: null,
```
Comment (tunetheweb) [inline src/lib/store.js:53]: Reverse this change before merging:

```suggestion
  cookiePreference: isProd ? null : 'accepts',
```

**Review by devnook — APPROVED** [review]

**Comment by tunetheweb** [comment]
Thanks @devnook . Will merge tomorrow (26th April) so can be online in case of any issues.

**Comment by devnook** [comment]
Thanks @tunetheweb !

**Pull request merged**